src/style.css

```diff
@@ -1 +1,38 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:hover {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-last-child(2),
+.stars--5 .stars__star:nth-last-child(2),
+.stars--5 .stars__star:last-child,
+.stars .stars__star:first-child:not(.stars--0 .stars__star),
+.stars
+  .stars__star:nth-child(2):not(.stars--0 .stars__star, .stars--1 .stars__star),
+.stars
+  .stars__star:nth-child(3):not(
+    .stars--0 .stars__star,
+    .stars--1 .stars__star,
+    .stars--2 .stars__star
+  ) {
+  background-image: url(/src/images/star-active.svg);
+}
```

Comment:
The selector logic for applying the active star image is overly complex and may not work as intended. The use of `:not()` pseudo-classes combined with multiple class selectors can lead to specificity issues. Consider simplifying the logic or testing thoroughly to ensure the correct stars are highlighted based on the rating.